src/components/Camera/StandardCamera.js

```diff
@@ -167,105 +174,195 @@ const StandardCamera = ( ): Node => {
+      : `${cameraOptionsClassName} m-[12.5px]` );
+    switch ( takePhotoOptions.flash ) {
+    case "on":
+      name = "flash-on";
       testID = "flash-button-label-flash";
       accessibilityLabel = t( "Flash-button-label-flash" );
       break;
-    case "flash-off":
+    case "off":
+      name = "flash-off";
       testID = "flash-button-label-flash-off";
       accessibilityLabel = t( "Flash-button-label-flash-off" );
       break;
     default:
       break;
     }
     return (
-      <Avatar.Icon
+      <Pressable
+        className={flashClassName}
+        onPress={toggleFlash}
+        accessibilityRole="button"
         testID={testID}
         accessibilityLabel={accessibilityLabel}
-        size={40}
-        icon={icon}
-        style={{ backgroundColor: colors.gray }}
-      />
+        accessibilityState={{ disabled: false }}
+      >
+        <INatIcon
+          name={name}
+          color={colors.white}
+          size={20}
+        />
+
+      </Pressable>
     );
   };
 
-  const checkmarkClass = "w-[40px] h-[40px] my-auto";
-
-  return (
+  const renderSmallScreenCameraOptions = () => (
     <>
-      <View className="flex-1 bg-black">
-        <StatusBar barStyle="light-content" />
-        <PhotoPreview
-          photoUris={cameraPreviewUris}
-          setPhotoUris={setCameraPreviewUris}
-          savingPhoto={savingPhoto}
-          screenBreakpoint={breakpoint}
+      { hasFlash && (
+        renderFlashButton()
+      ) }
+      <Pressable
+        className={`absolute bottom-[18px] right-[18px] ${cameraOptionsClassName}`}
+        onPress={flipCamera}
+        accessibilityLabel={t( "Camera-button-label-switch-camera" )}
+        accessibilityRole="button"
+        accessibilityState={{ disabled: false }}
+      >
+        <INatIcon
+          name="rotate"
+          color={colors.white}
+          size={20}
+          testID="camera-button-label-switch-camera"
         />
-        <View className="relative flex-1">
-          {device
-            && <CameraView device={device} camera={camera} orientation={imageOrientation} />}
-          <FadeInOutView savingPhoto={savingPhoto} />
-        </View>
-        <View className={`flex-row justify-between w-${footerWidth} mb-4 px-4`}>
-          {hasFlash ? (
-            <Pressable onPress={toggleFlash} accessibilityRole="button">
-              {takePhotoOptions.flash === "on"
-                ? renderFlashButton( "flash-on" )
-                : renderFlashButton( "flash-off" )}
-            </Pressable>
-          ) : (
-            <View />
-          )}
+      </Pressable>
+    </>
+  );
+
+  const checkmarkClass = "bg-inatGreen rounded-full h-[40px] w-[40px] justify-center items-center";
+
+  const renderLargeScreenCameraOptions = () => {
+    const buttonSpacing = ( imageOrientation.toLowerCase() === "portrait" )
+      ? "space-y-[40px]" : "space-y-[32px]";
+    return (
+      // eslint-disable-next-line max-len
```

Comment:
Instead of disabling the rule, you could do

```javascript
const cameraOptionsClassName = "bg-black/50 w-[40px] h-[40px] justify-center "
    + "items-center rounded-full";
```

or even better,

```javascript
const cameraOptionsClassName = [
    "bg-black/50",
    "h-[40px]",
    "items-center",
    "justify-center",
    "rounded-full",
    "w-[40px]"
  ].join( " " );
```

In general, try work with the linting rules instead of disabling them. They're almost all there for a reason, in this case to maintain the sanity of your fellow developers so we can all quickly and easily see all the values in this list of class names.